### ISSUE TITLE
Error on command use

### ISSUE THREAD

**Issue by mmagoo13** [Flibio/EconomyLite#33]
Im getting the following error when using the /econ give command

```
22.02 14:31:09 [Server] pool-2-thread-9/INFO An internal error has occurred!
22.02 14:31:09 [Server] pool-2-thread-9/ERROR [economylite]: null
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at java.lang.Thread.run(Unknown Source)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at java.util.concurrent.FutureTask.run(Unknown Source)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at org.spongepowered.common.scheduler.SchedulerBase.lambda$startTask$0(SchedulerBase.java:183)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at io.github.flibio.economylite.utils.commands.BaseCommandExecutor.lambda$execute$0(BaseCommandExecutor.java:69)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at io.github.flibio.economylite.commands.balance.BalanceCommand.run(BalanceCommand.java:70)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at io.github.flibio.economylite.impl.economy.LiteEconomyService.getOrCreateAccount(LiteEconomyService.java:64)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at io.github.flibio.economylite.impl.PlayerServiceCommon.setBalance(PlayerServiceCommon.java:76)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: 	at io.github.flibio.economylite.utils.sql.SqlManager.executeUpdate(SqlManager.java:114)
22.02 14:31:09 [Server] pool-2-thread-9/INFO [STDERR]: [io.github.flibio.economylite.utils.sql.SqlManager:executeUpdate:120]: java.lang.NullPointerException
22.02 14:31:09 [Server] pool-2-thread-9/ERROR [economylite]: com.zaxxer.hikari.pool.HikariPool$PoolInitializationException: Failed to initialize pool: IO Exception: "java.net.UnknownHostException: node326.chi.mcprohosting.com: node326.chi.mcprohosting.com: No address associated with hostname" [90028-192]
```

**Comment by Flibio** [comment]
Have you changed any configuration option anywhere? Are you running local or MySQL storage?

**Comment by mmagoo13** [comment]
No I have not changed a thing and I have mysql disabled

**Comment by Flibio** [comment]
Does this happen every time the command is run or was this a one-time occurrence?

**Comment by mmagoo13** [comment]
Every time it was run

**Comment by Flibio** [comment]
Could you please try to run this development build and see if the error persists?

~~Link Removed~~

**Comment by mmagoo13** [comment]
i have added that and tested it and i am still getting the error

**Comment by Flibio** [comment]
Could you please send your entire `economylite` folder in your configuration folder? I'm going to try and reproduce the error.

**Comment by mmagoo13** [comment]
https://www.dropbox.com/sh/udgj2m006lzvk0y/AAAxWRJIntPwWdRE_JuqL-FEa?dl=0

**Comment by Flibio** [comment]
Thanks, I'll investigate more tomorrow.

**Comment by mmagoo13** [comment]
quick random question is using the command from console supported?

**Comment by Flibio** [comment]
Yes it should work.

**Comment by Flibio** [comment]
Could you please try this version of EconomyLite and send the entire server startup log?

~~Link Removed~~

**Comment by mmagoo13** [comment]
http://pastebin.com/c6gLf3KL

**Comment by mmagoo13** [comment]
do you need the fml-server-latest log too?

**Comment by mmagoo13** [comment]
if so thats https://www.dropbox.com/s/t4nhveg1b65ybxb/fml-server-latest.log?dl=0

**Comment by Flibio** [comment]
Could you please try to remove LuckPerms temporarily and run the command? When you do please send `fml-server-latest.log` file.

**Comment by mmagoo13** [comment]
Still getting the same error unfortunately

**Comment by Flibio** [comment]
Okay, but can you send the `fml-server-latest.log` when running the command without LuckPerms?

**Comment by mmagoo13** [comment]
sure here you go
https://www.dropbox.com/s/t4nhveg1b65ybxb/fml-server-latest.log?dl=0

**Comment by Flibio** [comment]
Thanks, can you please try this version now? If it fails can you please send `fml-server-latest.log` with it?

http://downloads.flibio.net/EconomyLite-5.0.0-2.10.0-dev3.jar

**Comment by Flibio** [comment]
This should be fixed in `EconomyLite v2.10.1`.
